framework/test/mocha/unit/application/node/utils/config.js

```diff
@@ -1,142 +0,0 @@
-/*
- * Copyright © 2019 Lisk Foundation
- *
- * See the LICENSE file at the top-level directory of this distribution
- * for licensing information.
- *
- * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
- * no part of this software, including this file, may be copied, modified,
- * propagated, or distributed except according to the terms contained in the
- * LICENSE file.
- *
- * Removal or modification of this copyright notice is prohibited.
- */
-
-'use strict';
-
-/* eslint-disable */
-
-// This file is not used any more will be replaced completely with https://github.com/LiskHQ/lisk/issues/2976
-// const config = require('../../../../../../src/modules/chain/helpers/config');
-const { ZSchema } = require('../../../../../../src/application/validator');
-
-// This file is not used any more
-// eslint-disable-next-line
-describe.skip('helpers/config', () => {
```

Comment:
do we have some tests covering this configuration helper?

Reply:
we don't have this script anymore =)

Reply:
Individual cases are covered in separate unit tests for keywords.